vllm/distributed/kv_transfer/kv_connector/v1/offloading_connector.py

```diff
@@ -502,3 +657,69 @@ def yield_req_data(
         cached_reqs.new_block_ids,
         (req_id in cached_reqs.resumed_req_ids for req_id in cached_reqs.req_ids),
     )
+
+
+class OffloadPromMetrics(KVConnectorPromMetrics):
+    def __init__(
+        self,
+        vllm_config: VllmConfig,
+        metric_types: dict[type[PromMetric], type[PromMetricT]],
+        labelnames: list[str],
+        per_engine_labelvalues: dict[int, list[str]],
+    ):
+        super().__init__(vllm_config, metric_types, labelnames, per_engine_labelvalues)
+
+        counter_offload_block_queries = self._counter_cls(
+            name="vllm:offload_connector_block_queries",
+            documentation="Number of blocks queried by Offloading KV Connector.",
+            labelnames=labelnames,
+        )
+        self.counter_offload_block_queries = self.make_per_engine(
+            counter_offload_block_queries
+        )
+        counter_offload_block_hits = self._counter_cls(
+            name="vllm:offload_connector_block_hits",
+            documentation="Number of blocks queried and hit by Offloading"
+            " KV Connector.",
+            labelnames=labelnames,
+        )
+        self.counter_offload_block_hits = self.make_per_engine(
+            counter_offload_block_hits
+        )
+
```

Comment:
I think these field are redundant with respect to:
`vllm:external_prefix_cache_hits`
`vllm:external_prefix_cache_queries`.